<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CloudMaskingDockWidgetBase</class>
 <widget class="QDockWidget" name="CloudMaskingDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>875</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cloud Masking</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-5</y>
         <width>610</width>
         <height>889</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox_MTL">
          <property name="title">
           <string>Landsat Metadata File (MTL)</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QWidget" name="widget_3" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLineEdit" name="lineEdit_PathMTL"/>
              </item>
              <item>
               <widget class="QPushButton" name="Btn_FindMTL">
                <property name="text">
                 <string>Browse</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="Btn_LoadMTL">
                <property name="text">
                 <string>Load</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_4" native="true">
             <property name="font">
              <font>
               <pointsize>9</pointsize>
              </font>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="KLed" name="kled_LoadedMTL">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="state">
                 <enum>KLed::Off</enum>
                </property>
                <property name="look">
                 <enum>KLed::Flat</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_LoadedMTL">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_Filters">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="title">
           <string>Filters to apply</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="1">
            <widget class="QFrame" name="frame__BlueBand">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" name="gridLayout_7">
              <item row="0" column="0">
               <widget class="QCheckBox" name="checkBox_BlueBand">
                <property name="text">
                 <string>Blue Band</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QWidget" name="widget_BlueBand" native="true">
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <item>
                  <widget class="QWidget" name="widget_BlueBand_01" native="true">
                   <layout class="QHBoxLayout" name="horizontalLayout_5">
                    <item>
                     <widget class="QLabel" name="label_2">
                      <property name="text">
                       <string>Threshold</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QSlider" name="horizontalSlider_BB">
                      <property name="maximum">
                       <number>100</number>
                      </property>
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QDoubleSpinBox" name="doubleSpinBox_BB">
                      <property name="decimals">
                       <number>1</number>
                      </property>
                      <property name="maximum">
                       <double>100.000000000000000</double>
                      </property>
                      <property name="singleStep">
                       <double>0.100000000000000</double>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QFrame" name="frame_QCflags">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" name="gridLayout_9">
              <item row="0" column="0">
               <widget class="QCheckBox" name="checkBox_QCflags">
                <property name="text">
                 <string>Quality control flags</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QWidget" name="widget_QCflags" native="true">
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <item>
                  <widget class="QCheckBox" name="checkBox_4">
                   <property name="text">
                    <string>QC 01</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="checkBox_5">
                   <property name="text">
                    <string>QC 02</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="5" column="1" rowspan="3">
            <widget class="QFrame" name="frame">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <property name="lineWidth">
              <number>0</number>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QPushButton" name="Btn_processMask">
                <property name="text">
                 <string>Process mask</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widget" native="true">
                <layout class="QVBoxLayout" name="verticalLayout_7">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="margin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_processMaskStatus">
                   <property name="font">
                    <font>
                     <pointsize>9</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>TextLabel</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QProgressBar" name="progressBar">
                   <property name="font">
                    <font>
                     <pointsize>9</pointsize>
                    </font>
                   </property>
                   <property name="value">
                    <number>0</number>
                   </property>
                   <property name="invertedAppearance">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QFrame" name="frame_FMask">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" name="gridLayout_8">
              <item row="0" column="0">
               <widget class="QCheckBox" name="checkBox_FMask">
                <property name="text">
                 <string>FMask</string>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QWidget" name="widget_FMask" native="true">
                <layout class="QVBoxLayout" name="verticalLayout_2">
                 <item>
                  <widget class="QWidget" name="widget_FMask_01" native="true">
                   <layout class="QGridLayout" name="gridLayout_5">
                    <item row="1" column="1">
                     <widget class="QSlider" name="horizontalSlider_CP">
                      <property name="minimum">
                       <number>-50</number>
                      </property>
                      <property name="maximum">
                       <number>100</number>
                      </property>
                      <property name="singleStep">
                       <number>1</number>
                      </property>
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="2">
                     <widget class="QDoubleSpinBox" name="doubleSpinBox_CP">
                      <property name="decimals">
                       <number>1</number>
                      </property>
                      <property name="minimum">
                       <double>-50.000000000000000</double>
                      </property>
                      <property name="singleStep">
                       <double>0.100000000000000</double>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="0">
                     <widget class="QLabel" name="label">
                      <property name="text">
                       <string>Cloud probability</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QWidget" name="widget_FMask_02" native="true">
                   <layout class="QHBoxLayout" name="horizontalLayout_2">
                    <item>
                     <widget class="QLabel" name="label_3">
                      <property name="text">
                       <string>QA bands:</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QCheckBox" name="checkBox_2">
                      <property name="text">
                       <string>Cloud</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QCheckBox" name="checkBox_3">
                      <property name="text">
                       <string>Shadown</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QCheckBox" name="checkBox">
                      <property name="text">
                       <string>Snow</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <spacer name="horizontalSpacer">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>40</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_SaveApply">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="title">
           <string>Save and apply</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QWidget" name="widget_SaveApply_01" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Select the mask layer</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="select_MaskLayer">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="frame">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_4">
                <property name="text">
                 <string>Save mask</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_SaveApply_02" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Select the data layer</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="select_Data">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_5">
                <property name="text">
                 <string>Apply mask to data</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLed</class>
   <extends>QWidget</extends>
   <header>kled.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox_FMask</sender>
   <signal>clicked(bool)</signal>
   <receiver>widget_FMask</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_BlueBand</sender>
   <signal>clicked(bool)</signal>
   <receiver>widget_BlueBand</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>370</y>
    </hint>
    <hint type="destinationlabel">
     <x>132</x>
     <y>456</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_QCflags</sender>
   <signal>clicked(bool)</signal>
   <receiver>widget_QCflags</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>482</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>576</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>groupBox_SaveApply</sender>
   <signal>clicked(bool)</signal>
   <receiver>widget_SaveApply_01</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>714</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>740</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>groupBox_SaveApply</sender>
   <signal>clicked(bool)</signal>
   <receiver>widget_SaveApply_02</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>700</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>796</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
